core/src/it/scala/org/ensime/intg/BasicWorkflow.scala

```diff
@@ -245,13 +245,13 @@ class BasicWorkflow extends EnsimeSpec
           )
 
           // expand selection around 'val foo'
-          project ! ExpandSelectionReq(fooFile, 215, 215)
+          project ! ExpandSelectionReq(fooFile, 290, 290)
```

Comment:
PC implementation of `expand-selection` is not 100% equivalent to scalariform.
For example `val f\*cursor here*\oo = new Foo()` scalariform would expand to just the name `foo`, while the smallest tree is the whole `ValDef`.